modules/Settings/Picklist/models/Module.php

```diff
@@ -104,15 +111,25 @@ public function addPickListValues($fieldModel, $newValue, $rolesSelected = [])
 	 * @param string                        $oldValue
 	 * @param string                        $newValue
 	 * @param int                           $id
+	 * @param string                        $description
 	 *
 	 * @return bool
 	 */
-	public function renamePickListValues($fieldModel, $oldValue, $newValue, $id)
+	public function renamePickListValues($fieldModel, $oldValue, $newValue, $id, $description = '')
 	{
 		$db = App\Db::getInstance();
 		$pickListFieldName = $fieldModel->getName();
 		$primaryKey = App\Fields\Picklist::getPickListId($pickListFieldName);
-		$result = $db->createCommand()->update($this->getPickListTableName($pickListFieldName), [$pickListFieldName => $newValue], [$primaryKey => $id])->execute();
+		$tableName = $this->getPickListTableName($pickListFieldName);
+		$newData = [$pickListFieldName => $newValue];
+		$descriptionColumnExist = $this->checkDescriptionColumn($db, $tableName);
+		if (!empty($description) || $descriptionColumnExist) {
```

Comment:
if (!empty($description)) {

Reply:
jesli chcemy usunac przy edycji opis to samo !empty() nic nie da ale jesli istnieje tez kolumna to wtedy taka edycja sie powiedzie